src/views/ResultPage.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <div class="payform">
+        <img style="padding-right: 16px" :src="require(`@/assets/${pictPath}`)" alt="">
```

Comment:
markup and styles duplications are best candidates to be extracted to a separate component